gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,12 +1,6 @@
-#
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Do you think it's worth manually adding back the copyright notice in this properties file?

Reply:
I'm assuming this file was automatically generated/updated?  If so, there's little point in adding back the header as it'll disappear the next time it changes.

Reply:
ah, that's a good point... I don't remember manually adding this gradle change, so likely it's automatically generated :p